docs/reference/docs/reindex.asciidoc

```diff
@@ -1137,14 +960,211 @@ done
 ----------------------------------------------------------------
 // NOTCONSOLE
 
+[[docs-reindex-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=refresh]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=timeout]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=wait_for_active_shards]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=wait_for_completion]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=requests_per_second]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=scroll]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=slices]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=max_docs]
+
+[float]
+[[docs-reindex-api-response-body]]
+==== {api-response-body-title}
+
+//////////////////////////
+[source,console]
+--------------------------------------------------
+POST /_reindex?wait_for_completion
+{
+  "source": {
+    "index": "twitter"
+  },
+  "dest": {
+    "index": "new_twitter"
+  }
+}
+--------------------------------------------------
+// TEST[setup:twitter]
+
+//////////////////////////
+
+The JSON response looks like this:
+
+[source,console-result]
+--------------------------------------------------
+{
+  "took": 639,
+  "timed_out": false,
+  "total": 5,
+  "updated": 0,
+  "created": 5,
+  "deleted": 0,
+  "batches": 1,
+  "noops": 0,
+  "version_conflicts": 2,
+  "retries": {
+    "bulk": 0,
+    "search": 0
+  },
+  "throttled_millis": 0,
+  "requests_per_second": 1,
+  "throttled_until_millis": 0,
+  "failures": [ ]
+}
+--------------------------------------------------
+// TESTRESPONSE[s/: [0-9]+/: $body.$_path/]
+
+`took`::
+
+The total milliseconds the entire operation took.
+
+`timed_out`::
+
+This flag is set to `true` if any of the requests executed during the
```

Comment:
Usually include the data type in the response parms:
https://github.com/elastic/docs/blame/master/shared/api-ref-ex.asciidoc#L123